airbyte-integrations/connectors/source-datadog/source_datadog/manifest.yaml

```diff
@@ -137,13 +137,35 @@ definitions:
       path: "monitor"
     retriever:
       $ref: "#/definitions/retriever"
+      paginator:
+        type: "DefaultPaginator"
+        page_size_option:
+          type: "RequestOption"
+          inject_into: "request_parameter"
+          field_name: "page_size"
+        pagination_strategy:
+          type: "PageIncrement"
+          page_size: "{{ config['max_records_per_request'] }}"
+          start_from_page: 0
```

Comment:
If that was a string value, would the dependency PR still be required @dlecocq?

Reply:
If we set it to an int of our choosing, then they would be decoupled. I did try with `"500"` and the like, and it still complains. It has to be a literal yaml int in order for it to work.

FWIW, this is not a hill that I'd die on - if we wanted to hard code it for now, I think that's a sensible approach.